packages/itwinui-css/package.json

```diff
@@ -29,30 +29,34 @@
   ],
   "dependencies": {},
   "devDependencies": {
+    "@itwin/itwinui-icons-elements": "0.2.0",
     "autoprefixer": "^10.4.1",
     "backstopjs": "~6.0.4",
     "html-minifier": "^4.0.0",
+    "npm-run-all": "^4.1.5",
     "postcss": "^8.4.5",
     "postcss-discard-comments": "^5.0.1",
     "postcss-scss": "^4.0.3",
     "sass-embedded": "^1.49.9",
+    "serve": "^14.0.1",
     "stylelint": "^14.2.0",
-    "stylelint-config-sass-guidelines": "^9.0.1"
+    "stylelint-config-sass-guidelines": "^9.0.1",
+    "vite": "^3.0.0"
   },
   "scripts": {
     "build": "yarn clean && yarn build:scss && yarn build:css && node ../shared/copyrightLinter.js --fix css/* && yarn build:html",
     "build:scss": "cpx \"./src/**/*.scss\" ./scss",
     "build:css": "node ./scripts/generateCss.js src css",
-    "build:html": "node ./scripts/generateHtml.js backstop/tests backstop/minified",
-    "build:watch": "yarn build && concurrently -n scss,html -k \"yarn build:watch:scss\" \"yarn build:watch:html\"",
+    "build:html": "vite build backstop/tests --config vite.config.ts",
     "build:watch:scss": "chokidar \"src/**/*\" -c \"yarn build:scss && yarn build:css\"",
-    "build:watch:html": "chokidar \"backstop/tests/**/*\" -c \"yarn build:html\"",
     "createComponent": "node ./scripts/createComponent.js",
     "clean": "rimraf ./css ./scss ./backstop/results/bitmaps_test",
-    "dev": "yarn build:watch",
+    "dev": "concurrently -n scss,html -k \"yarn build:watch:scss\" \"yarn vite backstop/tests --config vite.config.ts\"",
```

Comment:
I couldnt figure from the scripts and configs...
If we run `yarn dev`, do we use `serve` anywhere? if yes, where this is defined? Cause i see only `preview` using it, but cant see we run `preview` while runnig `yarn dev`.

Reply:
`serve` is used in `yarn preview` which is used in `yarn test`

Reply:
amm... and it is used to only show up test results?
Cant we use `vite` same as with backstop html folder?

Reply:
aaa, ok.. now I see.. it is hosting backstop minified files, so tests could access those...?

Reply:
yeah, we need to run a server now because module imports are not allowed by simply visiting an html file.